server/src/main/java/org/apache/iotdb/db/service/metrics/FileMetrics.java

```diff
@@ -65,6 +65,7 @@ public void bindTo(AbstractMetricService metricService) {
   }
 
   private void bindTsFileMetrics(AbstractMetricService metricService) {
+    TS_FILE_METRIC_MANAGER.setMetricService(metricService);
```

Comment:
A little bit strange. We use TS_FILE_METRIC_MANAGER to manage metrics and add metric service into TS_FILE_METRIC_MANAGER. From my perspective, I think we can use TS_FILE_METRIC_MANAGER to manage related value, and use Metric Set to manage metric(get value from TS_FILE_METRIC_MANAGER)